README.md

```diff
@@ -114,7 +129,7 @@ Your query here
 5. How many offices are in the state of Colorado (United States).
 
 ```
-Your query here
+SELECT COUNT(*) list_of_office FROM offices WHERE state_id = 8;
```

Comment:
I think this should use a subquery pointing to "Colorado" instead of accessing directly to its `id`